components/inputs/input-select-styles.js

```diff
@@ -92,4 +92,21 @@ export const selectStyles = css`
 		color: transparent;
 		text-shadow: 0 0 0 #000000;
 	}
+
+	@media (prefers-contrast: more) {
+		.d2l-input-select,
+		.d2l-input-select:disabled,
+		.d2l-input-select:hover:disabled,
+		.d2l-input-select:focus:disabled {
+			background-image: url("data:image/svg+xml;base64,PHN2ZyB3aWR0aD0iMTEiIGhlaWdodD0iNyIgdmlld0JveD0iMCAwIDExIDciIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyI+CiAgPHBhdGggZD0iTTEgMmw0LjUgNE0xMCAyTDUuNSA2IiBzdHJva2U9IndoaXRlIiBzdHJva2Utd2lkdGg9IjIiIGZpbGw9Im5vbmUiIGZpbGwtcnVsZT0iZXZlbm9kZCIgc3Ryb2tlLWxpbmVjYXA9InJvdW5kIi8+Cjwvc3ZnPg==");
```

Comment:
Since this is just modifying the SVG to use a white stroke, a different high contrast scheme that sets a different `color` other than white won't work. For example the Windows "Dessert" theme:

<img width="197" alt="Screenshot 2025-05-07 at 9 41 38 AM" src="https://github.com/user-attachments/assets/6237d928-b84d-4e62-a879-f0653d96bd2b" />

I think we're safer to just disable our custom styles for `select`, checkboxes and radios by removing `appearance: none` in high contrast scenarios.